apps/05Mastermind.js

```diff
@@ -89,10 +82,21 @@ function mastermind(guess) {
 
 function getPrompt() {
     prompt.get(['guess'], function (error, result) {
-        console.log( mastermind(result['guess']) );
+        mastermind(result['guess']);
         if (!win) {
```

Comment:
Hey, it wasn't passing the mocha test, but it did detect a win if you played it. I moved the 'You guessed it!' message up to the mastermind function and it is now passing all tests!